src/lib/render.ts

```diff
@@ -51,6 +50,17 @@ const renderDistractorBlocks = (settings: ParsonsSettings): Cash => {
   return distractorBlockContainer
 }
 
+const getCodeBlocks = (code: string, isDistractors: boolean): string => {
+  const lines = code.split('\n')
+  const regex = /(.*?)\s*#distractor\s*$/;
+  if (isDistractors) {
+    return lines.filter(line => !line.search(regex))
+        .map(item => item.replace('#distractor',''))
```

Comment:
https://github.com/js-parsons/js-parsons/blob/master/parsons.js#L802

```#distractor``` should be removed only from line end